snprc_ehr/resources/queries/study/SPF_Results.js

```diff
@@ -0,0 +1,7 @@
+var console = require("console");
+var LABKEY = require("labkey");
+
+function beforeInsert(row, errors) {
+    row.objectid = row.objectid || LABKEY.Utils.generateUUID().toUpperCase();
+    // console.log("SPF_Results.js:  row.id = " + row.id + "row.objectid: " + row.objectid)
```

Comment:
Do we need to leave this commented console.log here?

Reply:
I got it in one of the later commits

Reply:
I have a bad habit of leaving them in place after I use them.